defaults/main.yml

```diff
@@ -9,108 +9,221 @@
 #
 # .. include:: includes/all.rst
 
-# .. Roundcube source and deployment [[[1
+# .. Packages and installation [[[
+#
+# -----------------------------
+#   Packages and installation
+# -----------------------------
+
+# .. envvar:: roundcube__required_php_packages [[[
+#
+# List of PHP packages required by Roundcube.
+# Refer to the `official Roundcube documentation <https://github.com/roundcube/roundcubemail/wiki/Install-Requirements>`__ for details.
+roundcube__required_php_packages:
+  # Included in the base install:
+  # - 'dom'
+  # - 'iconv'
+  # - 'openssl'
+  # - 'session'
+  # - 'sockets'
+  # - 'xml'
```

Comment:
I'm not sure how that will be parsed by `yaml2rst`. Perhaps a better way would be to move the list of included packages to the comment above, and make it an inline list instead of one entry for a line, which should make the comment smaller. Ditto for other instances.